packages/eslint-plugin/README.md

```diff
@@ -160,6 +160,7 @@ Then you should add `airbnb` (or `airbnb-base`) to your `extends` section of `.e
 | [`@typescript-eslint/no-require-imports`](./docs/rules/no-require-imports.md)                             | Disallows invocation of `require()`                                                                                                                 |                    |          |                   |
 | [`@typescript-eslint/no-this-alias`](./docs/rules/no-this-alias.md)                                       | Disallow aliasing `this`                                                                                                                            |                    |          |                   |
 | [`@typescript-eslint/no-type-alias`](./docs/rules/no-type-alias.md)                                       | Disallow the use of type aliases                                                                                                                    |                    |          |                   |
+| [`@typescript-eslint/no-unnecessary-condition`](./docs/rules/no-unnecessary-condition.md)                 | Prevents conditionals where the type is always truthy or always falsy                                                                               |                    |          | :thought_balloon: |
```

Comment:
This was supposed to be auto-generated by the `npm run docs` command, but I couldn't get that to run.  (It failed with "cannot find eslint-docs" and when I did `npx eslint-docs` it was looking in the wrong folder for rules), so I did this by hand.

Reply:
yeah we used to use a tool called `eslint-docs`, but when we migrated the repo to typescript, that tool stopped working.

We have a validation script to make sure the docs are correct, but not a generation script for it.